src/qt/rpcconsole.cpp

```diff
@@ -311,23 +313,66 @@ void RPCConsole::setClientModel(ClientModel *model)
         ui->peerWidget->setColumnWidth(PeerTableModel::Address, ADDRESS_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Subversion, SUBVERSION_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Ping, PING_COLUMN_WIDTH);
+        ui->peerWidget->horizontalHeader()->setStretchLastSection(true);
 
         // create context menu actions
-        QAction* disconnectAction = new QAction(tr("&Disconnect Node"), this);
+        QAction* disconnectAction   = new QAction(tr("&Disconnect Node"), this);
+        QAction* banAction1h        = new QAction(tr("&Ban Node for 1 hour"), this);
```

Comment:
You are repeating some translation text, perhaps there is an efficient way to reduce that?